src/qt/overviewpage.cpp

```diff
@@ -180,7 +180,6 @@ void OverviewPage::handleOutOfSyncWarningClicks()
 
 OverviewPage::~OverviewPage()
 {
-    if(timer) disconnect(timer, SIGNAL(timeout()), this, SLOT(coinJoinStatus()));
```

Comment:
Since line 167 is connecting with `this` as a receiver, the signal will be disconnected once `this` is destroyed. So no need to disconnect here.